RecoCTPPS/TotemRPLocal/python/totemRPLocalReconstruction_cff.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+# geometry
+from Geometry.VeryForwardGeometry.geometryRP_cfi import *
+XMLIdealGeometryESSource.geomXMLFiles.append("Geometry/VeryForwardData/data/RP_Garage/RP_Dist_Beam_Cent.xml")
```

Comment:
@ianna 
is it possible to make XMLIdealGeometryESSource DDCompactView with a separate label, which will be useful only for this CTPPS code?
this will avoid interference with other temporary/hardcoded modifications like 2017 geometry.
(we can disable ctpps in 2017 and later eras to make 2017 workflows run, but a more general solution would be to have a separate geometry "sub-universe")